website/docs/reference/datasources/querying-mongodb/README.md

```diff
@@ -245,7 +245,24 @@ people
 { $inc: { score: 1 } }
 ```
 
-`Limit`: The dropdown is used to configure if this delete command should act upon a single document or if this command should delete all the matching documents according to the query.
+`Limit`: The dropdown is used to configure if this delete command should act upon a **single document** or if this command should delete **all the matching documents** according to the query.
+
+MongoDB's multi update feature does not support replacement style updates. This means that you cannot replace the entire document, but rather, you can only update a single field.
```

Comment:
I don't think the user will understand this piece of info. In fact, it might actually confuse them. All we need to specify is that the Update query updates only one matching document when the limit is set to Single Document. When the Limit is set to all the matching documents, it updates all documents that match the query criteria

Reply:
updated the limit sentence

Reply:
@sumitsum can you do the technical review?

Reply:
LGTM